tasks/installer.py

```diff
@@ -79,13 +79,22 @@ def build(
 @task
 def build_linux_script(
     ctx,
+    output_bin=None,
     signing_key_id=None,
 ):
     '''
     Builds the linux script that is used to install the agent on linux.
     '''
+    script_path = os.path.join(BIN_PATH, "setup.sh")
+    signed_script_path = os.path.join(BIN_PATH, "setup.sh.asc")
     amd64_path = os.path.join(BIN_PATH, "bootstrapper-linux-amd64")
     arm64_path = os.path.join(BIN_PATH, "bootstrapper-linux-arm64")
+    if output_bin:
+        script_path = output_bin
+        signed_script_path = output_bin + ".asc"
+        amd64_path = output_bin + "-bootstrapper-linux-amd64"
+        arm64_path = output_bin + "-bootstrapper-linux-arm64"
```

Comment:
I suppose this 2 files need to be cleaned up after the base64 encode, otherwise they'd still end up being packaged no?

Reply:
I changed the logic a bit, I'm only uploading the `.sh` now.